app/javascript/react/components/OwlIcon.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import Blathers from '../../../assets/images/blathers_icon.png'
```

Comment:
heads up, this might work in development, but you will have issues with it in heroku/production.

We will let this merge, but be wary, you might have to add this image to amazon S3 and link to the image from there